www.foia.gov/_sass/layouts/_sidebar.scss

```diff
@@ -67,8 +76,12 @@
 .sidebar_content {
   flex-basis: auto;
   flex-grow: 6;
-  //approx 1 column + gutter in WDS grid at size
   @media screen and (min-width: $medium-screen) {
+    margin-left: 4%;
```

Comment:
Hmm, we should be sticking to the WDS grid as much as possible. This is getting into some strange territory 🤷‍♀️